### PR TITLE
Add Flatpak download links

### PR DESCRIPTION
**Pull request by kjarosh**
Supersedes https://github.com/ruffle-rs/ruffle-rs.github.io/pull/156

Merge after https://github.com/flathub/flathub/pull/5473

Screenshots after:
<p>
<image width="200" src="https://github.com/user-attachments/assets/6c281333-e8ce-4567-833b-566bd8645488"/>
</p>

<image width="200" src="https://github.com/user-attachments/assets/a0364ec9-f867-47d2-bc30-deefabb07bb5"/>

### CODE REVIEW

File: src/app/downloads/config.tsx [hunk 1]
```diff
@@ -132,6 +137,15 @@ export const desktopLinks: DownloadLink[] = [
     isRecommended: true,
     isDeviceRelevant: (device) => device.desktop && device.mac,
   },
+  {
+    key: "flatpak",
+    shortName: "Flatpak",
+    longName: "Flatpak Application",
```
Comment (torokati44) [inline src/app/downloads/config.tsx:143]: They seem to consistently use the term "app", but it's basically the same thing (and we use all kinds of nouns here, specific to each platform/store).

**Review by torokati44 — APPROVED** [review]

**Comment by torokati44** [comment]
Another point to consider (sorry if it was already discussed on Discord) - is putting the "long names" on the buttons really better? I'm fine with both, just don't see the point to change it.

File: src/app/downloads/config.tsx (hunk 1 above)
Comment (torokati44) [reply src/app/downloads/config.tsx:143]: Actually, uuuhh.... I don't even know anymore...
https://docs.flatpak.org/en/latest/introduction.html#terminology

Comment (torokati44) [reply src/app/downloads/config.tsx:143]: On the same hand (bottom right corner):
image
image

**Comment by Doomsdayrs** [comment]
Inquiry, This does not seem to add the badge that #156 adds, is this intentional?

File: src/app/downloads/config.tsx (hunk 1 above)
Comment (kjarosh) [reply src/app/downloads/config.tsx]: Changed to "App"

**Review by torokati44 — APPROVED** [review]

**Pull request merged**